storage/go.mod

```diff
@@ -12,4 +12,5 @@ require (
 	google.golang.org/api v0.52.0
 	google.golang.org/genproto v0.0.0-20210805201207-89edb61ffb67
 	google.golang.org/grpc v1.39.1
+	google.golang.org/protobuf v1.27.1
```

Comment:
There are already dependencies in this `go.mod` that transitively bring in `google.golang.org/protobuf` (genproto and grpc), so I'm not sure this is an issue, but @codyoss please confirm.

Reply:
Yeah this dependency is already there because of how our code is structured today sadly. Tested by making a simple file:

```
package main

import (
	"context"

	"cloud.google.com/go/storage"
)

func main() {
	storage.NewClient(context.Background())
}
```

Then I ran `go mod vendor`. Sources for `google.golang.org/protobuf` are already present.